### ISSUE TITLE
Start track from middle of album and then continue playing tracks

### ISSUE THREAD

**Issue by d8f6** [vanilla-music/vanilla#488]
Hello. I am new to vanilla music. The issue I am having (not sure if there is already a way to do this): I would like to go to an album or all songs from an artist and play a track but in the now playing have a list like when you select play all as opposed to clicking a track and only one track plays and that's it. I tried poking around the options but I cannot seem to figure it out. Okay thanks for reading.

**Comment by adrian-bl** [comment]
Sorry, but i think i do not understand your request.

Could you try to rephrase your question? Maybe a screenshot of the dialog in question would also help me

**Comment by dodo3773** [comment]
So, in Artists tab I click an artist name. From there it takes me to Albums tab. Then if I either swipe or click album name it takes me to Tracks tab. Then when I click on a song in the Tracks tab it plays only a single song. What I would like is to click a song and it play the whole album. Does that make more sense?

**Comment by dodo3773** [comment]
Not necessarily play an album but play from the list I am looking at. If it still doesn't make sense I can try to take a screen shot but I'm not sure a picture would explain it very well.

**Comment by adrian-bl** [comment]
I think what you are looking for is longpress(on song to start) -> play all

This will play all songs you are seeing, starting at the selected item.

Is this the expected functionality?

**Comment by dodo3773** [comment]
Yes. Long press -> play all is what I want. Is there a way to make this the default on the track tab? I tried changing the default to play all on default action in library setting but then I cannot navigate at all. It just plays all songs in the library.

**Comment by adrian-bl** [comment]
You can always 'expand' a tab by clicking the '>' part of the row.

But adding an 'Expand or Play all' option in the library settings might be a good idea.

**Comment by dodo3773** [comment]
Maybe if the library section in the options was split into options for the 3 tabs? Probably be a pain to implement? Well, feature request in that case! ;)  Now that I understand that this cannot currently be done without a change in the code base should I close this issue?

**Comment by adrian-bl** [comment]
> Maybe if the library section in the options was split into options for the 3 tabs? 

Which 3 tabs?

**Comment by dodo3773** [comment]
Artists, Albums, Tracks

I forgot that there are 4 though (Playlists)

**Comment by adrian-bl** [comment]
Ah, no: i'm not going to implement an per-app tap-option: Adding a new 'library default action' must suffice. (given the fact that the functionality is already there in the context menu, so this can always be used)

**Comment by dodo3773** [comment]
Gotcha. Okay, going to close then.

**Comment by adrian-bl** [comment]
Why are you closing this?

I said that i'm going to implement a new library action in the settings (expand by default, play all in tracks view) - i just do not want to have per-tab options.

**Comment by dodo3773** [comment]
Oh okay cool. Look forward to it!

**Comment by SebiderSushi** [comment]
I'd find per-tab options very useful. For example i'd like to have Play set as default for artists and albums but Play all for tracks.
Also i could use a differenciation between Play all and Play filtered (as in Poweramp) since i often search for a Song and like to play it together with all my tracks instead of just the other filter results.

**Comment by lottanbettan** [comment]
I wish that the "longpress(on song to start) -> play all" would be present when browsing in the files tab aswell, that would be awsome!
Great app! Thanks.

**Comment by d8f6** [comment]
Closing as this issue was fixed a while back.
